mlkem/verify.h

```diff
@@ -72,4 +78,17 @@ ASSIGNS(OBJECT_UPTO(r, sizeof(int16_t)))
 ENSURES(*r == (b ? v : OLD(*r)));
 // clang-format on
 
+#else /* MLKEM_USE_NATIVE */
```

Comment:
I was expecting this to parallel existing native functions. That is, use `#if defined(MLKEM_USE_NATIVE_VERIFY)` as a guard, and if set, define `static inline` functions (same name as in the non-native case) that forward to the native functions, e.g.
```c
static inline void cmov_int16(int16_t *r, const int16_t v, const uint16_t b) 
{
    return cmov_int16_native(r, v, b);
}
```

Reply:
done.